src/main/java/redis/clients/jedis/BasicCommands.java

```diff
@@ -41,4 +41,6 @@
   String configResetStat();
 
   Long waitReplicas(int replicas, long timeout);
+
+  String readonly();
```

Comment:
Wouldn't it make sense to add this command to the `ClusterCommands` interface?. As `readonly` it's available only under clustering conditions.